python/cudf/cudf/io/parquet.py

```diff
@@ -139,26 +140,24 @@ def read_parquet(
         filepath_or_buffer, None, **kwargs
     )
     if compression is not None:
-        ValueError("URL content-encoding decompression is not supported")
+        raise ValueError("URL content-encoding decompression is not supported")
 
     if engine == "cudf":
-        df = libparquet.read_parquet(
+        return libparquet.read_parquet(
             filepath_or_buffer,
-            columns,
-            row_group,
-            skip_rows,
-            num_rows,
-            strings_to_categorical,
-            use_pandas_metadata,
+            columns=columns,
+            row_group=row_group,
```

Comment:
Missing `row_group_count=row_group_count`

Reply:
thanks for catching that. Just pushed the update.